src/FormsSubmit.vue

```diff
@@ -46,8 +46,45 @@ export default {
 		return {
 			form: loadState('forms', 'form'),
 			isLoggedIn: loadState('forms', 'isLoggedIn'),
+			isEmbedded: loadState('forms', 'isEmbedded', false),
 			shareHash: loadState('forms', 'shareHash'),
 		}
 	},
+
+	mounted() {
+		if (this.isEmbedded) {
+			// Communicate window size to parent window in iframes
+			const resizeObserver = new ResizeObserver(entries => {
+				this.emitResizeMessage(entries[0].target)
+			})
+			this.$nextTick(() => resizeObserver.observe(document.querySelector('.app-forms-embedded form')))
+		}
+	},
+
+	methods: {
```

Comment:
```suggestion
	destroyed() {
		unsubscribe('forms:last-updated:set')
	},

	methods: {
```

Unsubscribe the event.